Tools/autotest/arducopter.py

```diff
@@ -794,6 +818,23 @@ def fly_gcs_failsafe(self, side=60, timeout=360):
 
     # Tests all actions and logic behind the battery failsafe
     def fly_battery_failsafe(self, timeout=300):
+        ex = None
+        try:
+            self.test_battery_failsafe(timeout=timeout)
+        except Exception as e:
+            ex = e
+
+        self.set_parameter('BATT_LOW_VOLT', 0)
```

Comment:
Any reason not to use a context here rather than assuming this list of parameters?

Reply:
It was done like that so I did it the same way.
I can use the context